x-pack/plugins/transform/server/types.ts

```diff
@@ -5,14 +5,14 @@
  * 2.0.
  */
 
-import type { IRouter, CoreStart } from '@kbn/core/server';
```

Comment:
Can we keep the `type` exports here or is there a reason to remove them?

Reply:
Updated here [`3f64b1f` (#154736)](https://github.com/elastic/kibana/pull/154736/commits/3f64b1f53667da00f811cc2579eb31decf81fafb)